scripts/portal-flow.js

```diff
@@ -37,14 +38,14 @@ function createRecord(baseUrl, request, clientId, dataset, data) {
     pending: [recordUtils.generateRecord(data)]
   });
   return request.post({
-    url: urlFor(baseUrl, 'workorders'),
+    url: urlFor(baseUrl, dataset),
     body: payload,
     json: true
   }).then(() => data);
 }
 
-module.exports = function(runner, argv) {
-  return function(previousResolution) {
+module.exports = function portalFlow(runner, argv) {
+  return previousResolution => {
```

Comment:
from a call stack perspective might be good to name the returned function too, not only the main export.